matter_server/server/sdk.py

```diff
@@ -365,7 +374,7 @@ async def find_or_establish_case_session(
         if self._chip_controller is None:
             raise RuntimeError("Device Controller not initialized.")
 
-        node_logger = LOGGER.getChild(f"node_{node_id}")
+        node_logger = self.get_node_logger(LOGGER, node_id)
```

Comment:
Now it would be really awesome if we could somehow also redirect the chip sdk native logs that include the node ID (now in hex format) because that is the most annoying to keep track of in the jungle of log output. Guess that is one bridge too far ?

Reply:
Yeah I did think about that. A hacky method to do that would be to parse the log string. However, the log strings are not uniform (look a bit different at times), so this is potentially unreliable. Also, performance wise it isn't ideal as the logs would bubble up from the native side to the Python side (acquiring the GIL), to then only getting filtered out there.

Ideally the SDK would support node id log filtering as well, which we then could set similarly.

I think the second is a bit too much work, so maybe it is still worth doing the first one for now. But this would be a follow up PR.